API.md

```diff
@@ -0,0 +1,387 @@
+# FBInk Command Line App Manual
+
+## Synopsis
+
+* ```sh
+  fbink [OPTIONS] [STRING ...]
+  ```
+
+  Print `STRING`s on your device's screen.
+
+* ```sh
+  fbink [-WHf] [-k] --refresh [top=NUM,left=NUM,width=NUM,height=NUM]
+  ```
+
+  Refresh the screen as per your specification, without touching the framebuffer.
+
+* ```sh
+  fbink [-WHf] --cls [top=NUM,left=NUM,width=NUM,height=NUM]
```

Comment:
Ditto on the missing supported flags (not necessarily the same one, though. The listing in the printf wall should be accurate).

Reply:
The associated description is below the ---sh block

Reply:
Yep:

https://github.com/NiLuJe/FBInk/blob/94305d2d2b40983dee12b2a236dac573ebbd556f/fbink_cmd.c#L117

Reply:
The "abort early" notice is somewhat relevant, because it means it can't be combined with other actions. (e.g., printing a string or an image).

(The `without touching the framebuffer` mention for `--refresh` essentially means the same thing, too).

Reply:
In the synopsis numbers 2 and 3, that usage is not advertising a string or other options, so the abort early is out of scope.

But in synopsis number 1, all of the options are available, even ones that collide, as so the "abort early" notice below applies, and it's in the Options specifications.

I think this is still right as-is. And-or I'm missing something.

Reply:
> In the synopsis numbers 2 and 3, that usage is not advertising a string or other options, so the abort early is out of scope.
> 

Technically correct, although I'm always in favor of making those thing explicitly clear ;).

> But in synopsis number 1, all of the options are available, even ones that collide, as so the "abort early" notice below applies, and it's in the Options specifications.
> 

Yep, wasn't talking about 1. at all ;).

> I think this is still right as-is. And-or I'm missing something.

It's certainly not *wrong* ;).

The only really missing stuff is the flags listing for 3..

Reply:
/me screwed head back on. Okay, I think d9a0227f3ce0104d2fe9c171eca5cc916b546a26 is what we're looking for here...